server/app/routes/users/index.js

```diff
@@ -0,0 +1,39 @@
+
+var router = require('express').Router();
+var User = mongoose.model('User');
+
+router.get('/', (req, res, next) => {
+	User.find({})
+	.then((users) => {res.json(user)})
+	.catch(next);
+});
+
+router.post('/', (req, res, next) => {
+	User.create(req.body)
+	.then((user) => res.status(201).json(user))
+	.catch(next);
+});
+
+router.get("/:id", (req, res, next) => {
+	User.findById(req.params.id)
+	.then((user) => res.json(user))
+	.catch(next);
+});
+
+router.put('/:id', (req, res, next) => {
+	User.findOneAndUpdate({_id: req.params.id}, req.body, {new: true})
```

Comment:
Needs to account for possibility of update validation. [findOneAndUpdate docs](http://mongoosejs.com/docs/api.html#query_Query-findOneAndUpdate)